website/static/js/projectSettings.js

```diff
@@ -152,7 +152,7 @@ request.fail(function(xhr, textStatus, err) {
  * Pulls a random name from the scientist list to use as confirmation string
  *  Ignores case and whitespace
  */
-var getConfirmationCode = function(nodeType, isPreprint) {
+var getConfirmationCode = function(nodeType, isPreprint, nodeApiUrl=nodeApiUrl) {
```

Comment:
I don't believe the `nodeApiUrl=nodeApiUrl` is doing anything. Probably can be just `nodeApiUrl`